models/demos/wormhole/mamba/tt/mamba_block.py

```diff
@@ -50,15 +50,33 @@ def __init__(self, args: ModelArgs, device, configs, load_fn: Callable):
 
         # conv1d_weights (2E, 1, 4)->(2E, 1)->(2E, 1, 1)->(1, 1, 2E)
         conv1d_weight_name = "mixer.conv1d.weight"
-        self.conv1d_weights = []
+        self.conv1d_weights_prefill = []
         for i in range(4):
-            self.conv1d_weights.append(
+            self.conv1d_weights_prefill.append(
                 load_fn(
                     conv1d_weight_name,
-                    lambda x: x[:, :, i].transpose(-1, -2).repeat(self.batch_size, 1).unsqueeze(0).unsqueeze(0),
-                    postfix=f"{i}_{args.batch_size}",
+                    lambda x: x[:, :, i]
+                    .transpose(-1, -2)
+                    .repeat(self.configs["outer_dim"], 1)
+                    .unsqueeze(0)
+                    .unsqueeze(0),
+                    postfix=f"{i}_{self.configs['outer_dim']}",
                 )
             )
+        self.conv1d_weights_decode = []
+        for i in range(4):
+            self.conv1d_weights_decode.append(
```

Comment:
Actually, no need to maintain conv1d_weights for prefill. They have been handled inside mamba_conv